api/benefit/models/benefit.settings.json

```diff
@@ -59,6 +59,11 @@
     },
     "EligibilityCriteria_FR": {
       "type": "richtext"
+    },
+    "benefit_key": {
```

Comment:
Can benefit_key be renamed to BenefitKey for consistency

Reply:
Of Course